server/src/main/java/io/spine/server/integration/TransportFactory.java

```diff
@@ -30,25 +29,24 @@
  * Publish-Subscriber Channel pattern.</a>
  *
  * @author Alex Tymchenko
+ * @author Dmitry Ganzha
  */
 @SPI
 public interface TransportFactory {
 
     /**
-     * Creates a {@link Publisher} for the messages of given class.
+     * Creates a {@link Publisher} for the messages of given channel identifier.
```

Comment:
`for the messages of given channel identifier.` 

Here and below does not make much sense. Messages cannot be of a `channel identifier`.